src/Plugin/Block/IslandoraBasicCollectionCollectionListing.php

```diff
@@ -13,7 +16,29 @@
  *   admin_label = @Translation("Islandora Collection Listing"),
  * )
  */
-class IslandoraBasicCollectionCollectionListing extends BlockBase {
+class IslandoraBasicCollectionCollectionListing extends BlockBase implements ContainerFactoryPluginInterface {
```

Comment:
May be nitpicky, but if most blocks for our use in Islandora require access to the configFactory to get the settings should we make a helper class in Islandora that we can extend for less future copypasta?